### PR TITLE
Feat/create user registration

### PR DESCRIPTION
**Pull request by mllrdev-glenn-henares**

### CODE REVIEW

File: src/core/services/user.service.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import router from "@/router"
+import axios from "axios"
+import { environment } from "@/environments/environment"
+
+
+export function loginUser(emailInput: string, passwordInput: string) {
```
Comment (mllrdev-carlo-medel) [inline src/core/services/user.service.ts]:
```suggestion
export function login(email: string, password: string) {
```

File: src/core/services/user.service.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        data: loginData
+    })
+        .then(function (response) {
+            console.log(response.data.isSuccessful)
+            if (response.data.isSuccessful) {
+                router.push("/home")
+            }
+            else if (!response.data.isSuccessful) {
+                router.push("/login")
+            }
```
Comment (mllrdev-carlo-medel) [inline src/core/services/user.service.ts]: Please trnasfer the page naviagtion in the page. All logic should just be the token management and pre-login stuffs.

File: src/core/services/user.service.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        url: environment.baseUrl + "/register",
+        data: signUpData
+    })
+        .then(function (response) {
+            console.log(response.data.isSuccessful)
+            if (response.data.isSuccessful) {
+                router.push("/login")
+            }
+            else if (!response.data.isSuccessful) {
+                router.push("/register")
```
Comment (mllrdev-carlo-medel) [inline src/core/services/user.service.ts]: Same.

File: src/environments/environment.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { IEnvironment } from "./IEnvironment";
+
+export const environment: IEnvironment = {
+    envName: "DEV",
+    production : false,
+    baseUrl :'http://localhost:8081/api/user'
```
Comment (mllrdev-carlo-medel) [inline src/environments/environment.ts]:
```suggestion
    baseUrl :'http://localhost:8081'
```

File: src/pages/Login.vue [hunk 5]
```diff
@@ -39,16 +41,24 @@ export default defineComponent({
...
     Toolbar,
   },
+  setup() {
+    const data = reactive({
+      email: "",
+      password: "",
+    });
+
+    const login = async () => {
+      await loginUser(data.email, data.password)
```
Comment (mllrdev-carlo-medel) [inline src/pages/Login.vue]:
```suggestion
      loginUser(data.email, data.password).then((data) => redirectToOtherPage() }
```

File: src/views/Home.vue [hunk 6]
```diff
@@ -39,7 +39,7 @@ import PurchaseAction from '@/core/enums/action.enum';
 import PurchaseStatus from '@/core/enums/status.enum';
 import { IonContent, IonPage, IonTitle, IonToolbar, IonGrid, IonRow, IonButton } from '@ionic/vue';
 import { defineComponent, ref } from 'vue';
-import IPurchaseOrder from '../core/interfaces/purchase-order';
+import IPurchaseOrder from '../core/interfaces/purchase-order.interface';
```
Comment (mllrdev-carlo-medel) [inline src/views/Home.vue]:
```suggestion
import IPurchaseOrder from 'src/core/interfaces/purchase-order.interface';
```

File: src/core/services/user.service.ts (hunk 1 above)
Comment (mllrdev-glenn-henares) [reply src/core/services/user.service.ts]: Done.

File: src/core/services/user.service.ts (hunk 2 above)
Comment (mllrdev-glenn-henares) [reply src/core/services/user.service.ts]: Done.

File: src/core/services/user.service.ts (hunk 3 above)
Comment (mllrdev-glenn-henares) [reply src/core/services/user.service.ts]: Done.

File: src/environments/environment.ts (hunk 4 above)
Comment (mllrdev-glenn-henares) [reply src/environments/environment.ts]: Done.

File: src/views/Home.vue (hunk 6 above)
Comment (mllrdev-glenn-henares) [reply src/views/Home.vue]: Done.

**Review by mllrdev-carlo-medel — APPROVED** [review]

**Pull request merged**